modules/widgets/twitter-timeline.php

```diff
@@ -371,6 +374,7 @@ class="widefat"
 				name="<?php echo $this->get_field_name( 'height' ); ?>"
 				type="number" min="200"
 				value="<?php echo esc_attr( $instance['height'] ); ?>"
+				<?php echo $instance['tweet-limit'] ? 'disabled' : ''; ?>
```

Comment:
Will this work in the customizer (Appearance > Customize), when you can enter a number of tweets and then save your changes without reloading the page?

Reply:
Great point! I actually didn't consider the customizer when writing this up, so I will have to revisit this sometime this week!